modules/monitoring-modifying-retention-time-and-size-for-prometheus-metrics-data.adoc

```diff
@@ -3,10 +3,20 @@
 // * monitoring/configuring-the-monitoring-stack.adoc
 
 :_content-type: PROCEDURE
-[id="modifying-retention-time-for-prometheus-metrics-data_{context}"]
-= Modifying the retention time for Prometheus metrics data
+[id="modifying-retention-time-and-size-for-prometheus-metrics-data_{context}"]
+= Modifying the retention time and size for Prometheus metrics data
 
-By default, the {product-title} monitoring stack configures the retention time for Prometheus data to be 15 days. You can modify the retention time to change how soon the data is deleted.
+By default, Prometheus automatically retains metrics data for 15 days.
+You can modify the retention time to change how soon data is deleted.
+You can also configure the maximum amount of disk space used by the retained metrics data.
+If this size limit is reached, the oldest data is deleted first until the disk space used by the data is again below the maximum.
```

Comment:
It's better append the following information
Retention takes into account only the block directories, it doesn't include the wal and chunks_head  directories.
also the retention policy is only applied when Prometheus cuts a new block which happens every 2 hours (and after the WAL contains at least 3h of data).

Reply:
Added two new items to the bulleted list:

```
* The retention settings apply only to data block directories. 
The settings are not used for data located in the `wal` and `chunks_head` directories.
* The retention settings are used only when Prometheus writes a new data block to a data block directory, which occurs every two hours and after the write-ahead log (WAL) has recorded at least three hours of data.
```